server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -762,6 +764,17 @@ public void execute(final String[] args) {
       if (opts.resetSecurity) {
         AccumuloServerContext context = new AccumuloServerContext(new ServerConfigurationFactory(HdfsZooInstance.getInstance()));
         if (isInitialized(fs)) {
+          if (!opts.forceResetSecurity) {
+            String secret = acuConf.get(Property.INSTANCE_SECRET);
+            ConsoleReader c = getConsoleReader();
+            String userEnteredSecret = c.readLine("WARNING: This will remove all users from Accumulo! If you wish to proceed enter the instance secret: ", '*');
```

Comment:
The prompt for the secret is similar to how GitHub asks you to type the name of the repo, to be sure you really want to delete it. It's not a security check. The user running this command already needs access to the server configuration file containing the secret. It's just a sanity check. Instead of the secret, it could prompt for the instance name or ID.